### DISCUSSION TITLE
OC-0.8.5 + 2013 / 2014 MacBook Pro SMBIOS

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [MultimediaLucario/Lenovo-ThinkPad-T440s#10] [Announcements]
![repository-open-graph-template new](https://user-images.githubusercontent.com/72415505/193061418-2920edc3-48f6-4397-b9da-e88fe475b626.png)

#
## Video output support for macOS Monterey is a Work In Progress. 🛠

This releases comes with two EFI folders, one that supports up to **macOS 11 Big Sur** which contains the 2013 13" MacBook Pro SMBIOS and one that supports **macOS 12 Monterey** which contains the 2014 15" MacBook Pro SMBIOS.

**Full Changelog**: https://github.com/MultimediaLucario/Lenovo-ThinkPad-T440s/compare/OC-0.8.3-IWFIX...OC-0.8.5

<hr /><em>This discussion was created from the release <a href='https://github.com/MultimediaLucario/Lenovo-ThinkPad-T440s/releases/tag/OC-0.8.5'>OC-0.8.5 + 2013 / 2014 MacBook Pro SMBIOS</a>.</em>

**Comment by 4mitabh** [comment]
Great!

This was the last missing feature. Everything else works perfectly!

**Reply by MultimediaLucario** [reply]
Thanks 😁
